plotbitrate.py

```diff
@@ -40,53 +40,58 @@
 import statistics
 import subprocess
 import sys
-from importlib import util
 from collections import OrderedDict
+from importlib import util
 from enum import Enum
 from typing import Callable, Union, List, IO, Iterable, Optional, Dict, Tuple, \
     Generator
+from frame import Frame
     
+
+class Color(Enum):
+    I = "red"
+    P = "green"
+    B = "blue"
+    AUDIO = "C2"
+    FRAME = "C0"
+
+
+class ConsoleColors:
```

Comment:
I see you have `END_COLOR` defined, but it doesn't appear to be used on any of the `sys.exit()` calls. I think this will end up coloring your shell prompt if you didn't have a colored prompt. For anyone reading the code adding a comment with what each color is might be nice.

Reply:
I was able to confirm that it colors my prompt red after an error message is produced. Since you call `sys.exit()` and number of times for error messages we should probably just make this a function for ease of use.